packages/http/src/index.ts

```diff
@@ -7,16 +7,18 @@ import route from './router';
 import { validateInput, validateOutput } from './validator';
 export * from './types';
 export * from './getHttpOperations';
+export * from './mocker/errors';
+export * from './router/errors';
 export * from './mocker/serializer/style';
 export { generate as generateHttpParam } from './mocker/generator/HttpParamGenerator';
 
-import { IHttpConfig, IHttpRequest, IHttpResponse, PickRequired, PrismHttpComponents } from './types';
+import { IHttpConfig, IHttpRequest, IHttpResponse, PickRequired, PrismHttpComponents, IHttpProxyConfig } from './types';
 
 export const createInstance = (
-  defaultConfig: IHttpConfig,
-  components?: PickRequired<Partial<PrismHttpComponents>, 'logger'>,
+  defaultConfig: IHttpConfig | IHttpProxyConfig,
```

Comment:
Please take a look at the comment on `http/src/types.ts`.

Reply:
5ecf360ca253ce09a6668e2c107e02ef5862133f